zuul.d/project-templates.yaml

```diff
@@ -711,18 +713,59 @@
 - project-template:
     name: ansible-collections-kubernetes-core-units
     check:
-      jobs: &ansible-collections-kubernetes-core-units-jobs
+      jobs:
         - build-ansible-collection
         - ansible-test-sanity-docker
         - ansible-test-sanity-docker-stable-2.9
         - ansible-test-sanity-docker-stable-2.10
         - ansible-test-sanity-docker-stable-2.11
         - ansible-test-units-kubernetes-core-python37
         - ansible-test-integration-kubernetes-core
-        - ansible-test-molecule-kubernetes-core
         - ansible-tox-linters
+        - molecule-test-splitter
+        - ansible-test-molecule-kubernetes-core-29_0
+        - ansible-test-molecule-kubernetes-core-29_1
+        - ansible-test-molecule-kubernetes-core-29_2
+        - ansible-test-molecule-kubernetes-core-29_3
+        - ansible-test-molecule-kubernetes-core-29_4
+        - ansible-test-molecule-kubernetes-core-210_0
+        - ansible-test-molecule-kubernetes-core-210_1
+        - ansible-test-molecule-kubernetes-core-210_2
+        - ansible-test-molecule-kubernetes-core-210_3
+        - ansible-test-molecule-kubernetes-core-210_4
+        - ansible-test-molecule-kubernetes-core-devel_0
+        - ansible-test-molecule-kubernetes-core-devel_1
+        - ansible-test-molecule-kubernetes-core-devel_2
+        - ansible-test-molecule-kubernetes-core-devel_3
+        - ansible-test-molecule-kubernetes-core-devel_4
```

Comment:
What is the idea of doing this?  This is pretty aggressive from a compute / testing POV, given each of these nodes is 4vcpu.  Are you hitting timeout issues?

Reply:
the 4vcpu configuration was originally with the full scenario execution, I will try that with less cpu, knowing that I will need at least 2 as this is running a k8s cluster.

Reply:
The better way here is to create a new nodeset, with 2 nodes. One for controller, the other appliance. Then you can boot a 1vcpu controller, and 2vcpu appliance (k8s).

We do this pattern of testing for network appliances, and it works well if you need to scale up.

That way, we can drop this complex logic, which is really only needed for limiting testing on aws

Reply:
yes this can be a solution, but we also trying here to speed up the execution, now the full molecule default scenario is running in more than 45 minutes
I will try this